packages/fluentui/docs/src/components/ComponentBestPractices.tsx

```diff
@@ -22,7 +22,7 @@ const ComponentBestPractices: React.FC<ComponentBestPracticesProps> = ({ doList,
   const updatedDontList: ShorthandCollection<ListItemProps> = [];
 
   if (_.size(doList) > 0) {
-    _.map(doList, element => {
+    _.map(doList, (element: string) => {
```

Comment:
From what I see the prop types aren't very well defined to be used like this. I would suggest that you change the prop types to be more accurate with what is actually used (I could not find a single shorthand usage):

```typescript 
  doList?: React.ReactNode[];
  dontList?: React.ReactNode[];
```
This would mean that you don't need the `string` cast which is actually wrong since there are react elements there too.